File: app/src/main/java/com/mgaetan89/showsrage/model/ShowsStat.java
package com.mgaetan89.showsrage.model;

import com.google.gson.annotations.SerializedName;

public class ShowsStat {
	@SerializedName("ep_downloaded")
	private int episodesDownloaded;
	@SerializedName("ep_snatched")
	private int episodesSnatched;
	@SerializedName("ep_total")
	private int episodesTotal;
	@SerializedName("shows_active")
	private int showsActive;
	@SerializedName("shows_total")
	private int showsTotal;

	public int getEpisodesDownloaded() {
		return this.episodesDownloaded;
	}

	public int getEpisodesMissing() {
		return this.episodesTotal - (this.episodesDownloaded + this.episodesSnatched);
	}

	public int getEpisodesSnatched() {
		return this.episodesSnatched;
	}

	public int getEpisodesTotal() {
		return this.episodesTotal;
	}

	public int getShowsActive() {
		return this.showsActive;
	}

	public int getShowsTotal() {
		return this.showsTotal;
	}
}

File: app/src/main/kotlin/com/mgaetan89/showsrage/model/ShowsStat.kt
package com.mgaetan89.showsrage.model

import com.google.gson.annotations.SerializedName
import io.realm.RealmObject

open class ShowsStat : RealmObject() {
	@SerializedName("ep_downloaded")
	open var episodesDownloaded: Int = 0
	@SerializedName("ep_snatched")
	open var episodesSnatched: Int = 0
	@SerializedName("ep_total")
	open var episodesTotal: Int = 0
	@SerializedName("shows_active")
	open var showsActive: Int = 0
	@SerializedName("shows_total")
	open var showsTotal: Int = 0

	val episodesMissing: Int
		get() = this.episodesTotal - (this.episodesDownloaded + this.episodesSnatched)
}
